### PR TITLE
Update Enum.md

### PR DESCRIPTION
**Pull request by tenshikiri**
Bumped to version 1.9.0 + a few extra spaces removed to match lines with en file.

### CODE REVIEW

File: lessons/el/basics/enum.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+  version: "1.9.0",
   title: "Enum",
   excerpt: """
   Μια συλλογή αλγορίθμων για την απαρίθμηση συλλογών.
   """
 }
 ---
 
-## Enum
+## Περίληψη
```
Comment (gemantzu) [inline lessons/el/basics/enum.md:10]: Don't change this :)

Comment (gemantzu) [inline lessons/el/basics/enum.md:10]: Don't change this :)

File: lessons/el/basics/enum.md [hunk 2]
```diff
@@ -31,10 +31,11 @@ at/3
 Με την παραπάνω έξοδο, είναι ξεκάθαρο ότι έχουμε ένα πολύ μεγάλο ποσό λειτουργιών, και για έναν καλό λόγο.
 Η απαρίθμηση βρίσκεται στον πυρήνα του συναρτησιακού προγραμματισμού, και η αξιοποίηση του μαζί με άλλα στοιχεία της Elixir, μπορεί να δώσει απίστευτη δύναμη σε έναν προγραμματιστή.
 
-Για να δείτε τις συναρτήσεις στο σύνολό τους επισκεφθείτε τα επίσημα έγγραφα του [`Enum`](https://hexdocs.pm/elixir/Enum.html).
-Για τεμπέλικη απαρίθμηση (lazy enumeration) χρησιμοποιήστε την ενότητα [`Stream`](https://hexdocs.pm/elixir/Stream.html).
+# Κοινές Συναρτήσεις
 
-### all?
+Για να δείτε τις συναρτήσεις στο σύνολό τους επισκεφθείτε τα επίσημα έγγραφα του [`Enum`](https://hexdocs.pm/elixir/Enum.html). Για τεμπέλικη απαρίθμηση (lazy enumeration) χρησιμοποιήστε την ενότητα [`Stream`](https://hexdocs.pm/elixir/Stream.html).
```
Comment (gemantzu) [inline lessons/el/basics/enum.md:36]: We seperate phrases with new lines.

File: lessons/el/basics/enum.md [hunk 3]
```diff
@@ -222,62 +222,60 @@ iex> Enum.uniq_by([%{x: 1, y: 1}, %{x: 2, y: 1}, %{x: 3, y: 3}], fn coord -> coo
...
- iex> Enum.map([1,2,3], &(&1 + 3))
- [4, 5, 6]
- ```
+```elixir
+iex> Enum.map([1,2,3], &(&1 + 3))
+[4, 5, 6]
+```
 
- Αυτό μπορεί περαιτέρω να καθαριστεί ορίζοντας την προηγουμένως ανώνυμη συνάρτηση που χρησιμοποιεί τον χειριστή Capture σε μία μεταβλητή και να τη καλλέσει από την `Enum.map/2`.
+Αυτό μπορεί περαιτέρω να καθαριστεί ορίζοντας την προηγουμένως ανώνυμη συνάρτηση που χρησιμοποιεί τον χειριστή Capture σε μία μεταβλητή και να τη καλλέσει από την `Enum.map/2`.
```
Comment (gemantzu) [inline lessons/el/basics/enum.md:249]: καλέσει (δες αν το έχουμε με 2 λ και αλλού)

**Review by gemantzu — CHANGES_REQUESTED** [review]

**Comment by doomspork** [comment]
@gemantzu do you want to bring this across the finish line?

**Pull request closed without merging**